docs/modules/idrac_system_erase.rst

```diff
@@ -0,0 +1,188 @@
+.. _idrac_system_erase_module:
+
+
+idrac_system_erase -- Erase system and storage components of the server
+=======================================================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Synopsis
+--------
+
+This module allows erasing System components such as iDRAC, BIOS, DIAG, etc., and Storage components such as PERC NV cache, Non-volatile memory, Cryptographic erase PD, etc. of the server.
+
+
+
+Requirements
+------------
+The below requirements are needed on the host that executes this module.
+
+- python \>= 3.9.6
+
+
+
+Parameters
+----------
+
+  component (True, list, None)
+    System and storage components to erase.
```

Comment:
List of system and storage components that can be deleted.
The following are the supported components:
- AllApps 
- BIOS
- CryptographicErasePD 
- DIAG 
- DPU 
- DrvPack 
- IDRAC
- LCData
- NonVolatileMemory 
- OverwritePD 
- PERCNVCache
- ReinstallFW 
- vFlash

Reply:
Done